include/StringProcessor.hpp

```diff
@@ -1,61 +1,65 @@
-#ifndef FTY_CONVERTER_STRINGPROCESSOR_H
-#define FTY_CONVERTER_STRINGPROCESSOR_H
+// SPDX-FileCopyrightText: 2020-2023 Ravil Dorozhinskii
+//
+// SPDX-License-Identifier: MIT
+
+#ifndef FTY_CONVERTER_STRINGPROCESSOR_HPP
+#define FTY_CONVERTER_STRINGPROCESSOR_HPP
 
 #include "FtyInternals.hpp"
 #include <regex>
 
 namespace fty {
 class StringProcessor {
-public:
+  public:
   /**
    * Checks whether the given line starts with 'Character'
    * @param String tests string
    * @param Character tests character
    * @return true if the string starts with 'Character'. Otherwise, false
    */
-  bool startsWith(const std::string &String, const char Character) {
+  static auto startsWith(const std::string& String, const char Character) -> bool {
```

Comment:
Why replace the return type with auto and then give it a trailing return type here?